src/Pages/Dashboard/Stats/Stats.js

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+import { UserContext } from "../../../Main/UserContext";
+import Wrapper from "../../../Shared/Wrapper/Wrapper";
+
+
+export default () => {
+  const userFootprint = useContext(UserContext).footprint[0];
+
+  return (
+    <Wrapper>
+      <div className="stats">
+        <div id="AMOandSIM">
+          <div id="AMO">
```

Comment:
Two things here:
- Try to use `classes` over `id`s - there can only be one element with the same id, so if you want to suddenly reuse some styling you will have to change this again
- Naming of things - I have no idea what these abbreviations mean, my guess would by "average monthly output", but the text says yearly. You could instead just call it `className="wrapper"` or something like that? :)

Reply:
Fair.